install-poetry.py

```diff
@@ -297,7 +297,15 @@ def make(cls, target: Path) -> "VirtualEnvironment":
             import venv
 
             builder = venv.EnvBuilder(clear=True, with_pip=True, symlinks=False)
-            builder.ensure_directories(target)
+            context = builder.ensure_directories(target)
+
+            if (
+                WINDOWS
+                and hasattr(context, "env_exec_cmd")
+                and context.env_exe != context.env_exec_cmd
+            ):
+                target = target.resolve()
```

Comment:
Are there other cases that MS Store Python where it could be useful to resolve the path?
I'm wondering if it may not be worth calling `resolve()` regardless of the context.

Reply:
I wasn't sure if this would have any downsides on any other platform, that's why I decided to only do this in this context.

Would be happy to simplify this if anyone can put in some knowledge here.

Reply:
I think let's take the cautious approach -- while in most environments I think unconditional resolve is okay, creative uses of symlinks by *nix users may cause unexpected breakage.